versions.tf

```diff
@@ -1,10 +1,10 @@
 terraform {
-  required_version = ">= 0.13.0"
+  required_version = ">= 1.0"
```

Comment:
do we have to move this pin to be greater than 1.0? I believe we only need to worry about the aws provider